lib/current-date.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const now = new Date();
+
+const currentDate = {
+  Y: now.getFullYear(),
+  D: now.getDate(),
+  M: now.getMonth() + 1,
+  h: now.getHours(),
+  m: now.getMinutes(),
+  s: now.getSeconds(),
+};
+
+module.exports = { currentDate };
```

Comment:
It is not really current, this would be cached at a time this file is first required and then always return the same date.

Reply:
Thanks, fixed